stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -4,13 +4,18 @@
 set -e
 
 LOG_DIR=${LOG_DIR:-/var/log}
-SDE_VERSION=${SDE_VERSION:-9.5.0}
+SDE_VERSION=${SDE_VERSION:-9.7.0}
```

Comment:
I don't think we should update the default version here/now. How about waiting for the flip in fabric-tna, then updating the default everywhere?